packages/core/src/api.ts

```diff
@@ -139,7 +139,14 @@ export class RetrieveMFATokenResponse extends Serializable {
 export class StartRegisterAuthenticatorParams extends Serializable {
     type: AuthType = AuthType.Email;
 
-    purposes: AuthPurpose[] = [AuthPurpose.Signup, AuthPurpose.Login, AuthPurpose.Recover];
+    purposes: AuthPurpose[] = [
```

Comment:
I'm not entirely sure if this is the right approach, but the client sent `0` as the purpose, which I mapped to the `MFAPurpose` indexes in the v3 server.